tests/integration/targets/lambda_layer/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/aws
+lambda_layer_info
+disabled
```

Comment:
Is there a reason this can't be added to CI?

Reply:
This has been disabled just to validate the PR before ``aws-terminator`` is updated with required permissions
And it seems that we are trying to reduce the number of unnecessary AWS calls in the module by using as many as possible unit testing